src/components/token/TokenAlerts.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TokenAlerts = (props) => {
+  const [token, setToken] = useState(props.token);
+
+  useEffect(() => {
+    setToken({banned: true});
+  }, [props.token]);
+
+  const verifiedAlert = () => {
+    if (!token.verified) {
+      return null;
+    }
+
+    return (
+      <div className="alert alert-info backup-alert" role="alert">
+        <i className="fa fa-check-circle mr-2" title="Token Verified"></i> This token is verified by Hathor team.
+      </div>
+    )
+  }
+
+  const bannedAlert = () => {
+    if (!token.banned) {
+      return null;
+    }
+
+    return (
+      <div className="alert alert-danger backup-alert" role="alert">
+        <i className="fa fa-exclamation-triangle mr-2" title="Token Verified"></i> This token was marked as scam by Hathor team.
```

Comment:
Maybe: "This token was banned and should not be trusted. <a>Learn more</a>". We should put Yan in the loop to check with our legal team what's the best language for this case.

Reply:
Ok, i'll get in touch with him

Reply:
Will check it. Anyway, I feel that saying the token was banned sounds weird, as it can still be exchanged freely. We might want to tweak the message. But let's wait for what the legal team before we do that.

Reply:
I've created an issue here to discuss the message: https://github.com/HathorNetwork/hathor-explorer/issues/92